notebooks/lst_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,1726 @@
+{
```

Comment:
<p>Why the specific values 0.05, 0.32 used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>I took those from an example in pyirf, no particular reason, its not needed for the function to set those values so I just remove them.</p>

Reply:
<p>This is what event display uses for the paranal array. Should not really relate to LST</p>